source/NVDAObjects/IAccessible/mozilla.py

```diff
@@ -83,7 +83,14 @@ def _get_detailsRole(self) -> Optional[controlTypes.Role]:
 			return None
 		for target in detailsRelations:
 			# just take the first for now.
-			return aria.normalizeDetailsRole(target.IAccessibleRole)
+			from .chromium import supportedAriaDetailsRoles

```

Comment:
Perhaps a note here that firefox may diverge in the future, and these values are just used out of convenience.

Reply:
I think this was missed?

Reply:
yep - looks like I forgot to commit the change. Fixed now.